how-to/workspace-platform-starter/client/src/framework/workspace/home.ts

```diff
@@ -220,7 +265,26 @@ async function onSelection(result: HomeDispatchedSearchResult): Promise<void> {
 				);
 			}
 		}
-	} else {
+	} else if (result.action.trigger === "user-action") {
 		logger.warn("Unable to execute result without data being passed");
 	}
 }
+
+/**
+ * By creating a dummy filters the UI always shows the filter row, which stops it
+ * bouncing as it adds and removes.
+ * @returns A dummy filter.
+ */
+function createDummyFilter(): CLIFilter {
+	return {
+		id: "app-searching",
+		title: "Tags",
```

Comment:
Should this be source instead of tags? As source in was means one or more integrations whereas tags are very specific to apps?

Reply:
Updated